api.planx.uk/webhooks/lowcalSessionEvents.ts

```diff
@@ -15,12 +15,14 @@ const createReminderEvent = async (req: Request, res: Response, next: NextFuncti
         status: 400,
         message: "Required value missing"
       });
-    const response = await createScheduledEvent({
-      webhook: "{{HASURA_PLANX_API_URL}}/send-email/reminder",
-      schedule_at: addDays(Date.parse(createdAt), (DAYS_UNTIL_EXPIRY - 7)),
-      payload: payload,
-      comment: `reminder_${payload.sessionId}`,
-    });
+    const response = await Promise.all(REMINDER_DAYS_FROM_EXPIRY.map((day: number) => (
+      createScheduledEvent({
+        webhook: "{{HASURA_PLANX_API_URL}}/send-email/reminder",
+        schedule_at: addDays(Date.parse(createdAt), (DAYS_UNTIL_EXPIRY - day)),
+        payload: payload,
+        comment: `reminder_${payload.sessionId}_${day}day`,
```

Comment:
As is, this comment will break our event cleanup procedures (eg PG functions `delete_lowcal_session_scheduled_events` and `delete_payment_request_scheduled_email_events`) which look for an exact string match on the comment with the prefix & sessionId only.

Would appreciate any opinions on possible approaches here - I see main options as: 
1. We don't care about unique comments on the events, but can determine which reminder it is (7 or 1 day) by it's scheduled date. I revert these changes to `comment` throughout, we don't have to replace existing PG functions.
2. We do care about unique comments on the events. We keep this structure, I `CREATE OR REPLACE` both PG functions to do fuzzy matching on comments using `reminder_sessionId%` so we can still easily add/subtract other reminder intervals in the future

Reply:
Yep, agree the comment is a really handy debugging context here :+1: Just wanted to sense check that direction before creating a migration